presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -0,0 +1,254 @@
+==========================================
+Presto Coordinator Kerberos Authentication
+==========================================
+
+The Presto server can be configured to enable Kerberos authentication over
+HTTPS for clients such as the :doc:`Presto CLI </security/cli>` or the various
+JDBC and ODBC drivers.
+
+.. warning::
+
+   Unauthenticated access over HTTP *must* be enabled because worker nodes
+   cannot yet be configured to connect to the coordinator using HTTPS or to
+   authenticate with Kerberos. It is the adminstrator's responsibility to
+   ensure that insecure access to the Presto coordinator is blocked for any
+   node that isn't a worker node.  This can be done by blocking access to the
+   Presto coordinator's HTTP port from nodes that are not workers.
+
+All of the Presto configuration changes are made on the coordinator. There are
+no changes required to the worker configuration to enable Kerberos
+authentication.
+
```

Comment:
Nope, worker nodes have to use unauthenticated HTTP access. The worker configuration doesn't need to be changed because they're configured to talk to the coordinator over unauthenticated HTTP by presto-admin by default :-/